diff --git a/node_modules/docker-compose-cdk/lib/.types-compat/ts3.9/resources/service-props/network-selection.d.ts b/node_modules/docker-compose-cdk/lib/.types-compat/ts3.9/resources/service-props/network-selection.d.ts
index 66d0a2f..e0d4b17 100644
--- a/node_modules/docker-compose-cdk/lib/.types-compat/ts3.9/resources/service-props/network-selection.d.ts
+++ b/node_modules/docker-compose-cdk/lib/.types-compat/ts3.9/resources/service-props/network-selection.d.ts
@@ -2,4 +2,5 @@ import type { INetwork } from '../network';
 export interface NetworkSelection {
     readonly network: INetwork;
     readonly ipV4Address?: string;
+    readonly aliases?: string[];
 }
diff --git a/node_modules/docker-compose-cdk/lib/resources/service-props/network-selection.d.ts b/node_modules/docker-compose-cdk/lib/resources/service-props/network-selection.d.ts
index 59c2c1a..9e16d0a 100644
--- a/node_modules/docker-compose-cdk/lib/resources/service-props/network-selection.d.ts
+++ b/node_modules/docker-compose-cdk/lib/resources/service-props/network-selection.d.ts
@@ -2,4 +2,5 @@ import type { INetwork } from '../network';
 export interface NetworkSelection {
     readonly network: INetwork;
     readonly ipV4Address?: `${number}.${number}.${number}.${number}`;
+    readonly aliases?: string[];
 }
diff --git a/node_modules/docker-compose-cdk/lib/resources/service.js b/node_modules/docker-compose-cdk/lib/resources/service.js
index 7b507a3..a08daf3 100644
--- a/node_modules/docker-compose-cdk/lib/resources/service.js
+++ b/node_modules/docker-compose-cdk/lib/resources/service.js
@@ -88,6 +88,7 @@ class Service extends ServiceBase {
                 return Object.assign(map, {
                     [mapping.network.name]: {
                         ipv4_address: mapping.ipV4Address,
+                        aliases: mapping.aliases,
                     },
                 });
             }, {}),
